configFile_LNGS.txt

```diff
@@ -59,14 +59,19 @@
 
 ### PMT waveform reconstruction
 'pmt_mode'              : 1,
+'board_pmt_channels'	: [1,2,3,4],	# Board channels used to save PMT.
```

Comment:
Now it works because if there is the slow digitizer the signals are in the same channels in both digitizers, but it could be different if the channels do not match.. Write this as a comment-memo so we do not forget

Reply:
Yes exactly. I thought about opening up that option but it seemed unnecessary at this point in time. I'll write down the comment.